phase2/ignition/vanilla/kubelet.service

```diff
@@ -15,6 +16,7 @@ ExecStart=/usr/bin/docker run \
         -v /var/run:/var/run:rw \
         -v /var/lib/docker/:/var/lib/docker:rw \
         -v /var/lib/kubelet/:/var/lib/kubelet:shared \
+        -v /var/log:/var/log:shared \
```

Comment:
Yes, something is smart enough to make it. I merged this PR locally (but without the `ExecStartPre` line) and deployed a cluster. Looks like logs are flowing into ES/Kibana now as expected and `/var/log/containers` was created on the nodes.